server/hedley/modules/custom/hedley_contact/hedley_contact.module

```diff
@@ -209,6 +209,17 @@ function hedley_contact_get_contacts_by_group($group, $language_code, $show_as_b
     });
   }
 
+  usort($contacts, 'hedley_contact_sort_by_full_name');
```

Comment:
Won't sort or asort be enough here?

Reply:
`usort` is required because it accepts a custom sorting function, [sort](http://php.net/manual/en/function.sort.php) doesn't.

Reply:
Yeah, I know - but won't using sort as is with an array of names give you the same result?

Reply:
I'm sorting a value inside the array, not the array keys, although, I can change the keys to be full names and just use `sort`

Reply:
Ok, leave is.